### ISSUE TITLE
`DispatchError` doesn't implement `Display`

### ISSUE THREAD

**Issue by frankli-dev** [paritytech/ink#987]
When I build Ink! contract, I get the following error

```
error[E0277]: `DispatchError` doesn't implement `Display`
  --> /contracts/uniswap_v2/lib.rs:13:1
   |
13 | #[ink::contract(env = crate::chain_extension::CustomEnvironment)]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `DispatchError` cannot be formatted with the default formatter
   |
   = help: the trait `Display` is not implemented for `DispatchError`
   = note: in format strings you may be able to use `{:?}` (or {:#?} for pretty-print) instead
   = note: this error originates in the macro `$crate::const_format_args` (in Nightly builds, run with -Z macro-backtrace for more info)
```

I get this error once I upgrade the ink version from =3.0.0-rc3 to =3.0.0-rc5

How can I fix this? please help

**Comment by Robbepop** [comment]
This should be fixed in ink! 3.0-RC6.

**Comment by frankli-dev** [comment]
yeah. It works with RC6. Thanks
